test/mocha.opts

```diff
@@ -1,3 +1,2 @@
 --compilers js:babel-core/register
---recursive
```

Comment:
That way tested directories, e.g. `test/screenshot_integration` are not included in `mocha` which is what we want.